runtime/development/src/lib.rs

```diff
@@ -2725,6 +2772,9 @@ impl_runtime_apis! {
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 			impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 
+			use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsicsBenchmark;
+			impl pallet_xcm::benchmarking::Config for Runtime {}
```

Comment:
FYI @wischli:

Independently if we go with a more complete solution or not, this fix by now the benchmark compilation and we can proceed with the upgrade pipeline